nanoDFT/nanoDFT.py

```diff
@@ -421,7 +423,6 @@ def nanoDFT_options(
         import mogli
         import imageio
         import matplotlib.pyplot as plt
-        opts.basis = "6-31G"
```

Comment:
I would keep it, so we don't change the logic of the code in this refactoring. You can update namedtuple in this way:
```
opts = opts._replace(basic="6-31G")
```

Reply:
Fixed, thanks!